com.microsoft.mrtk.uxcore/Interop/CanvasProxyInteractor.cs

```diff
@@ -11,9 +11,9 @@ namespace Microsoft.MixedReality.Toolkit.UX
     /// A simple proxy interactor which will select + hover things on our behalf, for canvas input.
     /// </summary>
     [AddComponentMenu("MRTK/UX/Canvas Proxy Interactor")]
-    public class CanvasProxyInteractor : XRBaseInteractor, IProxyInteractor
+    public class CanvasProxyInteractor : XRBaseInteractor, IProxyInteractor, IModeManagedInteractor
     {
-        protected HashSet<IXRInteractable> validTargets = new HashSet<IXRInteractable>();
+        protected HashSet<IXRInteractable> canvasValidTargets = new HashSet<IXRInteractable>();
```

Comment:
These aren't necessarily canvas objects, so `canvasValidTargets` is a little misleading.

Reply:
Fixed, the initial rename was due to collision with XRBaseControllerInteractor